questionpy_sdk/webserver/elements.py

```diff
@@ -0,0 +1,188 @@
+#  This file is part of QuestionPy. (https://questionpy.org)
+#  QuestionPy is free software released under terms of the MIT license. See LICENSE.md.
+#  (c) Technische Universität Berlin, innoCampus <info@isis.tu-berlin.de>
+
+from re import sub, Pattern
+from typing import List, Union, Annotated, Any, Optional
+from typing_extensions import TypeAlias
+from pydantic import Field, BaseModel
+
+from questionpy_common.elements import StaticTextElement, GroupElement, HiddenElement, RepetitionElement, \
+    SelectElement, RadioGroupElement, Option, CheckboxGroupElement, CheckboxElement, TextInputElement, FormSection
+# pylint: disable=unused-import
+from questionpy_common.elements import FormElement  # noqa: F401
+
+
+def replace(pattern: Pattern[str], replacement: str, string: str) -> str:
+    if not string:
+        return ""
+    return sub(pattern, f'{replacement}', string)
```

Comment:
Eigentlich sollte diese Funktion, insbesondere das `f'{replacement}'` nicht nötig sein. Es ist nötig aufgrund des letzten Kommentars, da du in `replacement` einen `int` durchschleifst, obwohl `str` als Hint dran steht. 

```suggestion
def replace(pattern: Pattern[str], replacement: str, string: str) -> str:
    if not string:
        return ""
    return sub(pattern, replacement, string)
```

mMn kannst du diese Funktion einfach durch `re.sub` ersetzen.